core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -114,6 +118,17 @@ public UnderFileSystemConfiguration setShared(boolean shared) {
   public UnderFileSystemConfiguration createMountSpecificConf(Map<String, String> mountConf) {
     UnderFileSystemConfiguration ufsConf = new UnderFileSystemConfiguration(mProperties.copy());
     ufsConf.mProperties.merge(mountConf, Source.MOUNT_OPTION);
+    String ufsConfigValue = ufsConf.get(PropertyKey.UNDERFS_HDFS_CONFIGURATION);
+    if (ufsConfigValue.startsWith("kv@")) {
```

Comment:
can we make `"kv@"` a constant?

Reply:
@gpang Yeah, I defined it in the PropertyKey.Name